cocotb/share/makefiles/simulators/Makefile.aldec

```diff
@@ -160,7 +159,6 @@ OLD_PATH := $(shell echo "$(PATH)" | sed 's/(/\\(/g' | sed 's/)/\\)/g' | sed 's/
 LIB_LOAD = PATH=$(MINGW_BIN_DIR):$(OLD_PATH):$(LIB_DIR)
```

Comment:
Possibly:
```suggestion
LIB_LOAD = PATH=$(MINGW_BIN_DIR):$(OLD_PATH)
```

Reply:
In the `PATH` on windows is it actually needed.

Reply:
makes sense, I now see the other simulators do the same thing.